Tasks/CmdLineV2/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -9,6 +9,8 @@
   "loc.input.label.workingDirectory": "Working Directory",
   "loc.input.label.failOnStderr": "Fail on Standard Error",
   "loc.input.help.failOnStderr": "If this is true, this task will fail if any errors are written to the StandardError stream.",
+  "loc.input.label.ignoreHostExpression": "Ignore HostException from Invoke-Expression cmdlet (works only on windows)",
```

Comment:
Users don't know anything about task internals (that CmdLine task calls Invoke-Expression inside). I would focus only on symptoms which customers may see, and avoid any details about task internals

Reply:
Fixed and added doc